authentication/account/service/user_blackbox_test.go

```diff
@@ -291,6 +304,9 @@ func (s *userServiceBlackboxTestSuite) TestListUsersToDeactivate() {
 		config.GetUserDeactivationInactivityPeriodFunc = func() time.Duration {
 			return 30 * 24 * time.Hour // 30 days
 		}
+		config.GetUserDeactivationInactivityNotificationPeriodFunc = func() time.Duration {
+			return 20 * 24 * time.Hour // 20 days
+		}
```

Comment:
Do we have a test for a following flow:
- User A is not active for N days.
- Check if User A is returned to be notified.
- Check if User A is not returned to be deactivated yet.
- User A does some activity.
- Check if User A is not returned to be notified.
- Check if User A is not returned to be deactivated.
- After notification period the user A still not among users to notify or deactivate.

Reply:
ok, let me add a flow to cover that case, although I believe that my sub tests already cover those cases, but independently

Reply:
I had a related question.. is the `deactivation_notification` value reset to null when there is user activity?

Reply:
@alexeykazakov test added in 9c3c1aa